### PR TITLE
fix now deployment scaling

### PR DESCRIPTION
**Pull request by IsaacHardy**
Fixes scaling issues for now deployment.

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings (in app AND in storybook)
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [x] Add a line to the changelog documenting the high-level change this PR introduces

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review the changelog entry

### CODE REVIEW

**Comment by conrad-vanl** [comment]
I think it's better then what we have, unless you think we should take scaling out entirely from our automated process and do it manually?

**Comment by IsaacHardy** [comment]
@conrad-vanl @richarddubay Found a command in the documentation that allows us to remove urls that are not aliased to our main domain. We've tested this and this seems to be the best solution. It builds, alias, then remove unused instances. Also, it makes every new build scaled to 1.

**Comment by conrad-vanl** [comment]
@IsaacHardy nice!! are you going to implement that change to this PR?

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,6 @@ jobs:
       before_deploy: npm install -g now
       deploy:
         - provider: script
-          script: cd api && now --public --token $NOW_TOKEN && now alias --token $NOW_TOKEN
```
Comment (conrad-vanl) [inline .travis.yml]: should the `now rm` be done _before_ the deploy? it seems like doing it after like this is just going to remove the instance you just deployed?

**Pull request merged**